.github/workflows/build.yml

```diff
@@ -230,6 +233,10 @@ jobs:
             base_path: ./bitwarden_license/src
             docker_repos: [bitwarden, bitwardenqa.azurecr.io]
             dotnet: true
+          - service_name: Scim
+            base_path: ./bitwarden_license/src
+            docker_repos: [bitwardenqa.azurecr.io]
```

Comment:
Would we not want to add this to our won `bitwarden` repo, or is that just because it's not yet in production?

Reply:
The `bitwarden` registry is Docker Hub. We can publish there as well if you would like. I don't see any problem publishing to Docker Hub.